mlflow/utils/autologging_utils/__init__.py

```diff
@@ -544,3 +544,22 @@ def should_log(self):
             )
 
     return _TrainingSession
+
+
+def temporarily_disable_autologging(name):
```

Comment:
we have an `mlflow.utils.autologging_utils.disable_autologging()` method. Can we add this functionality to that context manager? I.e. introduce an optional `name` parameter?

Reply:
I rethink on this. I think we don't need add `name` parameter for `mlflow.utils.autologging_utils.disable_autologging()`, the globally temporary disabled is fine, because mlflow only support single thread running, The case which another flavor code running in parallel won't happen.